src/ticket/Ticket.java

```diff
@@ -0,0 +1,28 @@
+package ticket;
+
+import datetimeutils.DateTimeUtils;
+
+public class Ticket {
+
+    public long fineID;
+    private final GovAuthorities authorityName;
+    public DateTimeUtils issueDate;
+    public Boolean isPaid;
+    public double fine;
+    private static final double UMA = 207.44;
+
+    public Ticket(long fineID, DateTimeUtils issueDate, GovAuthorities authorityName, Boolean isPaid) {
+        this.fineID = fineID;
+        this.issueDate = issueDate;
+        this.authorityName = authorityName;
+        this.isPaid = isPaid;
```

Comment:
Por default un ticket nace con un estado de no estar pagado, entonces es mejor que nazca como:
```suggestion
        this.isPaid = false;
```